backend/src/db.ts

```diff
@@ -1,12 +1,12 @@
-const { Client } = require('pg');
+import { Client } from 'pg';
 
-const config = JSON.parse(process.env.DB_CONNECT);
+const config: any = JSON.parse(process.env.DB_CONNECT);
```

Comment:
process.env.DB_CONNECT make this like string

and pass it to Client as {connectionString: process.env.DB_CONNECT}

and I guess it will be 'string' type